packages/api/config/lambdas.yml

```diff
@@ -156,35 +157,35 @@ CreateReconciliationReport:
 
 ScheduleSF:
   description: 'This lambda function is invoked by scheduled rules created via cumulus API'
-  handler: index.scheduler
+  handler: index.schedule
   timeout: 100
-  memory: 512
-  source: 'node_modules/@cumulus/api/dist/'
+  memory: 192
+  source: 'node_modules/@cumulus/api/dist/sfScheduler/'
   tables:
     - CollectionsTable
     - ProvidersTable
   namedLambdaDeadLetterQueue: true
 
 BulkDelete:
-  handler: index.bulkDeleteLambda
-  source: 'node_modules/@cumulus/api/dist/'
+  handler: index.handler
+  source: 'node_modules/@cumulus/api/dist/bulkDelete/'
 
 KinesisInboundEventLogger:
-  handler: index.logger
+  handler: index.handler
   timeout: 300
-  memory: 512
-  source: 'node_modules/@cumulus/api/dist/'
+  memory: 128
```

Comment:
What's the actual upper limit on inbound kinesis ingest records?     

*edit* answered that myself, it's small enough that shouldn't be a big deal..... however why is the OutboundEventLogger not similarly sized?

I want to be a little cautious about these as they're not backed by a DLQ (it's compliance logging, but still....)

Reply:
When I adjusted these, I based the memory allocation on how much memory each Lambda used during our integration tests.  We apparently invoke the InboundEventLogger during our tests, but not the OutboutEventLogger, so I adjusted the Inbound one but left Outbound the way it was.